.vscode/settings.json

```diff
@@ -1,9 +1,5 @@
 {
   "editor.formatOnSave": true,
   "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "[mdx]": {
-    "editor.defaultFormatter": null,
-    "editor.formatOnSave": false
-  },
```

Comment:
This should not be removed (or at least first discussed and in another PR dedicated to it).

Reply:
I do not know why we cannot use prettier to format mdx files as well. For me sometimes the formatting does not work in mdx and I think its due to this lines. VSCode tells me that it isnt sure what formatter to use with mdx files and then it acts all confuesed. I reverted the change for now though.